app/assets/stylesheets/govuk_publishing_components/components/_share-links.scss

```diff
@@ -83,3 +83,37 @@ $column-width: 9.5em;
     display: block;
   }
 }
+
+.gem-c-share-links--extra-spacing {
```

Comment:
I think given that this extra spacing is really only needed for the new icon set I'd suggest combining the two into a single option - I don't think we need them to be separate.

I'd also suggest this because the extra spacing option on the normal icons makes the text align to the bottom of the icons, which I'm not sure looks right.

Reply:
Thanks @andysellick - I've merged the two down to one now. Have pushed it as a separate commit but will squash eventually.